### PR TITLE
Fix Mate - Maintenance Service App - V-0 

### PR DESCRIPTION
**Pull request by Mofuhidy**
# ⚙🛠FixMate - Maintenance Service App ⚒🔨

## About Project 📖
**Fix Mate** is committed to making everyday life more convenient by connecting users with trusted service providers. It is a one-stop platform for finding, booking, and reviewing services, all from the comfort of your home. Users can communicate with service providers through the app, discussing specific requirements, asking questions, and confirming details.

## Key Features 🔑
- **Show all Experts**
- **Reserve an Expert**
- **Show my Reservation**
- **Add an Expert**
- **Delete an Expert**
- **Add a Specialization**

## Front End
<details>
  <summary>Client</summary>
  <ul>
    <li><a href="https://reactjs.org/">React</a></li>
    <li><a href="https://redux.js.org/">Redux</a></li>
  </ul>
</details>

### Changes Made ♻
- [x] Design the authentication (SignUp/Register, Login and Logout)
- [x] Create components
- [x] Used redux and store
- [x] Add images
- [x] Style according to the design by [Murat Korkmaz on Behance](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign)
- [x] Achieved Responsiveness      

## Useful links 🔗
✔ [Front-end Website](https://fixmate.netlify.app/)
✔ [Back-end Repository](https://github.com/anita00001/fix-mate-backend)
✔ [Kanban Board Link](https://github.com/users/anita00001/projects/5)
✔ [Api Url](https://fixmate.onrender.com/)
✔ API endpoints
   - api/v1/experts
   - api/v1/reservations
   - api/v1/specializations
✔ [API documentation](https://fixmate.onrender.com/api-docs/index.html)
## Future Enhancements
- [ ] **Delete Specializations**
- [ ] **Add Admin Role**

## Assets
- ER-Diagram
- Kanban Board Screenshot
- App Authentication
- App Home Page

#

## Best Regards,
@anita00001  @Mofuhidy  @Theodoraldo  @AnsarIbrahim

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Home from './components/Main/Home';
+
+function App() {
```
Comment (Meltrust) [inline src/App.js]: - [ ]  Please, kindly refactor this "App" functional component to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪


- [ ] OPTIONAL: Refactor the rest of the functions:

image

You can do a search on Vscode for "function" and correct as needed.

That way, all your functions will be up to standards. 👍

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @Mofuhidy,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:


-  Beautiful frontend ✔️ 



#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: src/App.js (hunk 1 above)
Comment (AnsarIbrahim) [reply src/App.js]: Solved Use ES6

**Review by Meltrust — APPROVED** [review]
#### Hi @Mofuhidy,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause

### To highlight

- All functions are now ES6✔️ 
- Great job✔️ 




Cheers and Happy coding!👏👏👏


------

**Pull request merged**